### PR TITLE
fix(*): tile display

### LINKED ISSUE #608: Empty map tails when return to same route
Hi flutter_map team! 

In examples, if you try to open same route the map widget show no tiles, until you trying to move or zoom map. See screen cast.

Any ideas how it can be fixed?

### LINKED ISSUE #636: Tiles are not shown if used in another instance of FlutterMap
For details look at the screen record from current version of example app.
To reproduce just start the example app and switch between 'MapController' and 'AnimatedMapController'. Unfortunately it’s blocking the app in my use case of FlutterMap because I push multiple instances of FlutterMap with `ineractive: false` to the navigation stack and you can just remove the 'grayscreen' by zooming in or out. Would be awesome to have a solution for that.

video-2020-05-28-16-56-41

Best Regards

### PR DESCRIPTION
**Pull request by ruizalexandre**
This PR fix **ONLY** fetch images !

-> Fetch image after adding in tiles dict

**I found an error on rotation present on version 9. 
Tile layer logic is broken, fix here. => https://github.com/johnpryan/flutter_map/pull/641**

🚀 Works fine on iOS/Android
🔄 Race condition fixed

Fix #636
Fix #608

### CODE REVIEW

File: lib/src/layer/tile_layer.dart [hunk 1]
```diff
@@ -327,16 +325,18 @@ class _TileLayerState extends State<TileLayer> with TickerProviderStateMixin {
         ),
       )..listen(_update);
     }
+
+    super.initState();
```
Comment (avioli) [inline lib/src/layer/tile_layer.dart:329]: Why is `super.initState` called last? Only `super.dispose` is advised to be called last.

Comment (ruizalexandre) [reply lib/src/layer/tile_layer.dart:329]: I'm following instruction from Dart/Flutter Plugin for VSCode only

```@override
  void initState() {
    // TODO: implement initState
    super.initState();
  }
```

Comment (ruizalexandre) [reply lib/src/layer/tile_layer.dart:329]: I’lol update this line to first line :)

**Comment by kuhnroyal** [comment]
Can you rebase this so this can get into a minor patch version.
The last version had a lot of fixes but this is till missing.

**Comment by catmorte** [comment]
hey! please do something!)

**Comment by ruizalexandre** [comment]
This week I’ll rebase all of this and update all my PRs

**Comment by ruizalexandre** [comment]
> Can you rebase this so this can get into a minor patch version.
> The last version had a lot of fixes but this is till missing.

Done :)

**Pull request closed without merging**